### DISCUSSION TITLE
Failed to load a collection

### DISCUSSION THREAD

**Discussion by xiaojuan8** [milvus-io/milvus#21964] [Q&A and General discussion]
I try to load a collection just created. But I wait for a long time, the collection was not loaded successfully. Kubernetes logs on querycoord node shows that I failed to load segments. Please help me on this issue. 

![querycoord fail](https://user-images.githubusercontent.com/74544828/216542095-b06cbd7c-9df8-4a68-be51-5261d07f4beb.png)

**Comment by xiaojuan8** [comment]
I modified querynode replicas from 1 to 2/4/8, loading collection fails all the time. But replicas = 1 always load collection successfully.

**Comment by yhmo** [comment]
Similar issue: https://github.com/milvus-io/milvus/issues/21196, this issue was fixed in the latest code(will be available on the next minor version v2.2.3).

Possible reason: the querynode was crashed and cannot serve.
Please collect the log of the querynode to create an issue on the [issues](https://github.com/milvus-io/milvus/issues) panel. Don't forget tell us your milvus version.
Try to restart the querynode and load again.

**Reply by xiaojuan8** [reply]
Milvus version: 2.2.0
but I checked querynodes. They are alive.
![milvus nodes](https://user-images.githubusercontent.com/74544828/216565402-3d2b255a-b0f9-47da-b2e5-4be8245da10f.png)

**Comment by yhmo** [comment]
Might be an unknown issue, please collect the log of query nodes and querycoord and create a new issue for us. Provide more information as possible as you can.

**Reply by xiaojuan8** [reply]
hello, I have create an issue : https://github.com/milvus-io/milvus/issues/21997
